### PR TITLE
feat (cli): set configuration file with flags, various small improvements

### PR DESCRIPTION
**Pull request by fsamin**

### CODE REVIEW

File: sdk/cli/cds/main.go [hunk 1]
```diff
@@ -5,7 +5,8 @@ import (
 	"os"
 
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
+
+	"path"
```
Comment (sguiheux) [inline sdk/cli/cds/main.go]: wrong place

File: sdk/cli/cds/login/login.go [hunk 2]
```diff
@@ -34,49 +44,61 @@ type config struct {
...
 		conf.Token = res.Token
 	} else {
 		conf.Password = string(password)
 	}
 
+	//Write conf in file
 	data, err := json.MarshalIndent(conf, " ", " ")
 	if err != nil {
 		sdk.Exit("Error: Cannot create config file (%s)\n", err)
 	}
```
Comment (yesnault) [inline sdk/cli/cds/login/login.go]: Warn if already exists?

File: sdk/cli/cds/main.go (hunk 1 above)
Comment (fsamin) [inline sdk/cli/cds/main.go]: indeed

File: sdk/cli/cds/login/login.go (hunk 2 above)
Comment (fsamin) [inline sdk/cli/cds/login/login.go]: yup yup

**Comment by fsamin** [comment]
plz squash and merge.

message should be : `feat (cli): set configuration file with flags, various small improvements`

**Comment by sguiheux** [comment]
LGTM

**Comment by proullon** [comment]
nice

**Comment by proullon** [comment]
nice

**Comment by proullon** [comment]
nice

**Comment by proullon** [comment]
nice

**Comment by proullon** [comment]
nice

**Comment by proullon** [comment]
nice

**Comment by proullon** [comment]
nice

File: sdk/cli/cds/login/login.go [hunk 3]
```diff
@@ -45,6 +45,16 @@ type config struct {
 func runLogin() {
 	conf := config{}
 
+	//Check if file exists
+	if _, err := os.Stat(sdk.CDSConfigFile); err == nil {
+		fmt.Printf("File %s exist, do you want to overwrite? [y/N]: ", sdk.CDSConfigFile)
```
Comment (yesnault) [inline sdk/cli/cds/login/login.go]: 's'...

**Pull request merged**